Cargo.toml

```diff
@@ -3,6 +3,13 @@ name = "bdk-cli"
 version = "0.1.0"
 edition = "2018"
 authors = ["Alekos Filini <alekos.filini@gmail.com>", "Riccardo Casatta <riccardo@casatta.it>"]
```

Comment:
I think you should add yourself here

Reply:
:pray: thanks guys, you did all the hard work but I'm happy to add my name and help keep the updates coming